server/src/frontend_api.rs

```diff
@@ -262,11 +265,52 @@ async fn post_enabled_features(
     )))
 }
 
+#[utoipa::path(
+context_path = "/api",
+responses(
+(status = 202, description = "Accepted client metrics"),
+(status = 403, description = "Was not allowed to post metrics"),
+),
+request_body = ClientMetrics,
+security(
+("Authorization" = [])
+)
+)]
 #[post("/proxy/client/metrics")]
+async fn post_proxy_metrics(
+    edge_token: EdgeToken,
+    metrics: Json<ClientMetrics>,
+    metrics_cache: Data<MetricsCache>,
+) -> EdgeResult<HttpResponse> {
+    let metrics = metrics.into_inner();
+
+    let metrics = from_bucket_app_name_and_env(
+        metrics.bucket,
+        metrics.app_name,
+        edge_token.environment.unwrap(),
+    );
+
+    metrics_cache.sink_metrics(&metrics);
+
+    Ok(HttpResponse::Accepted().finish())
```

Comment:
Is this code the same as for /fronted/client/metrics? I don't know if it's worth having a private method with this logic or if it's just fine having this duplication. Not a big deal TBH

Reply:
Yeah, this is the third time we duplicate it, so might be worth extracting to a common method. Thanks for giving me the push.